### ISSUE TITLE
[Design Doc] - Task Card Enhacement Design Doc In My-site

### ISSUE THREAD

**Issue by Pratiyushkumar** [Real-Dev-Squad/website-my#497]

**Comment by Ajeyakrishna-k** [comment]
> Also need to show assigned to name below the progress bar.

We are fetching only the logged in user's task, so I don't think assignee name and image is required in my-site.

**Comment by Pratiyushkumar** [comment]
> > Also need to show assigned to name below the progress bar.
> 
> We are fetching only the logged in user's task, so I don't think assignee name and image is required in my-site.

fixed

**Comment by Ajeyakrishna-k** [comment]
> > > Also need to show assigned to name below the progress bar.
> > 
> > 
> > We are fetching only the logged in user's task, so I don't think assignee name and image is required in my-site.
> 
> fixed

Please also update the milestones.

**Comment by Ajeyakrishna-k** [comment]
In the Figma provided I have two suggestions: 
1. Do we need info icon at the end of the dates? can't we just trigger the tooltip when hovered over the dates? 
2. Would `Request extension` be better text than `Raise extension request`


![Screenshot 2023-10-11 at 5 07 42 PM](https://github.com/Real-Dev-Squad/website-my/assets/98796547/623a8b84-3bc2-42b9-9715-d0b4274f7172)

**Comment by vinit717** [comment]
LGTM

**Comment by iamitprakash** [comment]
@Pratiyushkumar do we really need so many status types like smoke testing and all

**Comment by Pratiyushkumar** [comment]
> @Pratiyushkumar do we really need so many status types like smoke testing and all

No, we are removing currently those statuses that are not required, we will be adding those in the future but now we don't require them.

**Comment by iamitprakash** [comment]
> > @Pratiyushkumar do we really need so many status types like smoke testing and all
> 
> No, we are removing currently those statuses that are not required, we will be adding those in the future but now we don't require them.

whatever we don't need as of now remove that please

**Comment by iamitprakash** [comment]
Approved @Pratiyushkumar
